mqt/qecc/cc_decoder/test_square_octagon_color_code.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+from square_octagon_color_code import SquareOctagonColorCode
+
+
+def test_d3():
+    d3_layout = SquareOctagonColorCode(3)
```

Comment:
Similar to the other test file for the Hexagonal code, this should be a pytest fixture that is reused throughout the tests.